src/test/java/com/alphawallet/attestation/cheque/TestRedeemCheque.java

```diff
@@ -71,25 +71,25 @@ public void testSunshine() {
     try {
       PublicKey pk;
       System.out.println("Signed attestation:");
-      System.out.println(DERUtility.printDER(attestedCheque.getAtt().getDerEncoding(), "SIGNEABLE"));
+      System.out.println(DERUtility.toPEM(attestedCheque.getAtt().getDerEncoding(), "SIGNEABLE"));
```

Comment:
The tests now only print out the object reference and not the actual PEM file in human readable base 64.

Reply:
Thanks fixed.